e2e/charts/newrelic-infrastructure-k8s-e2e/templates/daemonset.yaml

```diff
@@ -54,12 +54,10 @@ spec:
             {{- end }}
           env:
             {{- if eq .Values.daemonset.clusterFlavor "Minikube" }}
-              {{- if and (ne .Values.daemonset.clusterVersion "v1.15.x") (ne .Values.daemonset.clusterVersion "v1.14.x") (ne .Values.daemonset.clusterVersion "v1.13.x") }}
+              {{- if semverCompare ">=1.16-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
What does the `-0` do?

Reply:
https://github.com/Masterminds/semver#working-with-prerelease-versions IIRC it includes the prereleases.